### ISSUE TITLE
[Accessibility] [Pagination] Basic Pagination > Page change status not announced by screen reader when using Previous/Next buttons.

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2025]
[A11Y-585](https://jira.cengage.com/browse/A11Y-585)

**Environment:**
Windows 11 / Chrome 137 / JAWS 2025
Windows 11 / Edge 136 / NVDA 2025
Mac 15 / Safari / VoiceOver

**Steps to Reproduce:**
1. Open the URL – https://react-magma.cengage.com/version/4.9.1/api/pagination/
2. Turn on a screen reader (JAWS/NVDA/VoiceOver).
3. Launch 'Pagination' component page from left navigation.
4. On the 'Pagination' page, under 'Basic Pagination'.
5. Use the Previous or Next button to change pages.
6. Observe whether any status message is announced by the screen reader.

**Actual Result:**
When the user changes the page using the Previous or Next button, no status message is announced by the screen reader. There is no indication of the current page or total number of pages.

Refer to Screencast: https://app.screencast.com/vHpBZWr6GZNF9

**Expected Result:**
When the user changes the page using the Previous or Next button, a status message should be announced, such as:
"[Title of the page] page `<n>` of `<m>` " (for example:- search results page 1 of 10.
where:
n = current page number
m = total number of pages

**User Impact:**
Screen reader users are left unaware of which page they are on after navigating, which can lead to confusion and difficulty in understanding the context of the content being viewed.

WCAG Guidelines:
4.1.3 Status Messages (Level AA)

**Bug Fix Recommendation:**
Use an ARIA live region (e.g., aria-live="polite") to announce the current page status dynamically when the page changes.

**Comment by silvalaura** [comment]
Solution: add aria-atomic to pagination component. We want to do this for both pagination types.

**Comment by nikitaorliak-cengage** [comment]
**Tested on:**
- V5: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/table/#pagination
- V4: https://next--upbeat-sinoussi-f675aa.netlify.app/api/table/#pagination

Everything works as expected. No bugs found.
